.eleventy.js

```diff
@@ -51,6 +56,55 @@ module.exports = function (config) {
   // Allow yaml to be used in the _data dir
   config.addDataExtension("yaml", contents => yaml.load(contents));
 
+  config.addFilter('contrastRatio', function(color1, color2) {
+    return contrastRatio(color1, color2);
+  });
+
+  // TODO: Where can we organize these functions so they're not in the main eleventy.js file?
```

Comment:
These files replicate the functionality provided by [the color contrast plugin](https://github.com/18F/brand/blob/8fa0a02e115286dace5301ebedc00932b4dd9328/_plugins/contrast_ratio.rb#L2).

I assume there's a better place to put these functions than the 11ty configuration file, but I didn't have a clear enough read on our bundling strategy to know where that should be — open to suggestions.

Reply:
I'm going to try #2, since that's how Handbook does it.